SocialCareCaseViewerApi.Tests/V1/IntegrationTests/MASH/UnAssignWorkerFromMashReferralTests.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.Tests.V1.Helpers;
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace SocialCareCaseViewerApi.Tests.V1.IntegrationTests.MASH
+{
+    [TestFixture]
+    public class UnAssignWorkerFromMashReferralTests : IntegrationTestSetup<Startup>
+    {
+        private SocialCareCaseViewerApi.V1.Infrastructure.MashReferral _existingDbReferral;
+
+        [SetUp]
+        public void Setup()
+        {
+            // Clear test database of any rows in the database
+            DatabaseContext.Teams.RemoveRange(DatabaseContext.Teams);
+            DatabaseContext.Workers.RemoveRange(DatabaseContext.Workers);
+            DatabaseContext.MashReferrals.RemoveRange(DatabaseContext.MashReferrals);
+
+            // Create existing referral and unrelated worker
+            var existingDbReferral = IntegrationTestHelpers.SaveMashReferralToDatabase(DatabaseContext, "CONTACT");
+            _existingDbReferral = existingDbReferral;
```

Comment:
You should be able to combine these two lines into one declaration I think 🤔 e.g
```c#
_existingDbReferral = IntegrationTestHelpers.SaveMashReferralToDatabase(DatabaseContext, "CONTACT");
```

Reply:
Copy paste from existing test ( will refactor that too )